docs/changelog.md

```diff
@@ -4,6 +4,18 @@ tocMaxDepth: 2
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.99 (2022-05-25)
+
+### Features
+
+- Added `response-contains-header` & `response-contains-property` built-in rules.
+- Added bundle option to keep url references.
+- Added new built-in rule `scalar-property-missing-example`.
```

Comment:
```suggestion
- Added three new built-in rules: `response-contains-header`, `response-contains-property`, `scalar-property-missing-example`.
- The `bundle` command supports a new option `--keep-url-references`. Use it to prevent Redocly CLI from resolving external URL references during bundling.
```